IPython/lib/lexers.py

```diff
@@ -230,9 +230,12 @@ class IPythonConsoleLexer(Lexer):
     # The regexps used to determine what is input and what is output.
     # The default prompts for IPython are:
     #
-    #     c.PromptManager.in_template  = 'In [\#]: '
-    #     c.PromptManager.in2_template = '   .\D.: '
-    #     c.PromptManager.out_template = 'Out[\#]: '
+    #    in           = 'In [#]: '
+    #    continuation = '   .D.: '
+    #    template     = 'Out[#]: '
+    #
+    # Where '#' is the input number and 'D' 
+    # is the number of digit of the input number. 
```

Comment:
D is dots matching the width of the input number

Also, we generally use the terms 'prompt number' or 'execution count', rather than input number.